### ISSUE TITLE
CIEDE feature results in ffmpeg crash

### ISSUE THREAD

**Issue by slhck** [Netflix/vmaf#1151]
I am using the latest ffmpeg build from https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz (v5.1.1) which bundles libvmaf 2.3.0.

When I run this command:

```bash
ffmpeg \
  -nostdin \
  -nostats \
  -y \
  -threads 0 \
  -r 25.0 -i /tmp/ref/ref-1280x720.mkv \
  -r 25.0 -i /tmp/dist/dist-854x480.mkv \
  -filter_complex \
  '[1][0]scale2ref=flags=bicubic[dist][ref];
  [dist]settb=AVTB,setpts=PTS-STARTPTS[distpts];
  [ref]settb=AVTB,setpts=PTS-STARTPTS[refpts];
  [distpts][refpts]libvmaf='"'"'model=path=/usr/local/share/model/vmaf_v0.6.1.json:log_path=/tmp/wbzljh9v-libvmaf.txt:log_fmt=json:n_threads=0:n_subsample=1:feature=name=ciede'"'"'' \
  -an -f null \
  /dev/null
```

I get the following ffmpeg output:

```
ffmpeg version N-63101-gc92edd969a-static https://johnvansickle.com/ffmpeg/  Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 8 (Debian 8.3.0-6)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-debug --disable-ffplay --disable-indev=sndio --disable-outdev=sndio --cc=gcc --enable-fontconfig --enable-frei0r --enable-gnutls --enable-gmp --enable-libgme --enable-gray --enable-libaom --enable-libfribidi --enable-libass --enable-libvmaf --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-librubberband --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libvorbis --enable-libopus --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libdav1d --enable-libxvid --enable-libzvbi --enable-libzimg
  libavutil      57. 36.101 / 57. 36.101
  libavcodec     59. 42.104 / 59. 42.104
  libavformat    59. 30.101 / 59. 30.101
  libavdevice    59.  8.101 / 59.  8.101
  libavfilter     8. 48.100 /  8. 48.100
  libswscale      6.  8.108 /  6.  8.108
  libswresample   4.  9.100 /  4.  9.100
  libpostproc    56.  7.100 / 56.  7.100
Input #0, matroska,webm, from '/tmp/ref/ref-1280x720.mkv':
  Metadata:
    ENCODER         : Lavf58.20.100
  Duration: 00:00:00.12, bitrate: 526 kb/s
  Stream #0:0: Video: h264 (High 4:4:4 Predictive), yuv444p(progressive), 1280x720 [SAR 1:1 DAR 16:9], 25 fps, 25 tbr, 1k tbn (default)
    Metadata:
      ENCODER         : Lavc58.35.100 libx264
      DURATION        : 00:00:00.120000000
Input #1, matroska,webm, from '/tmp/dist/dist-854x480.mkv':
  Metadata:
    ENCODER         : Lavf58.20.100
  Duration: 00:00:00.12, bitrate: 444 kb/s
  Stream #1:0: Video: h264 (High 4:4:4 Predictive), yuv444p(progressive), 854x480 [SAR 1:1 DAR 427:240], 25 fps, 25 tbr, 1k tbn (default)
    Metadata:
      ENCODER         : Lavc58.35.100 libx264
      DURATION        : 00:00:00.120000000
Stream mapping:
  Stream #0:0 (h264) -> scale2ref
  Stream #1:0 (h264) -> scale2ref
  libvmaf:default -> Stream #0:0 (wrapped_avframe)
Output #0, null, to '/dev/null':
  Metadata:
    encoder         : Lavf59.30.101
  Stream #0:0: Video: wrapped_avframe, yuv444p(tv, progressive), 1280x720 [SAR 1281:1280 DAR 427:240], q=2-31, 200 kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.42.104 wrapped_avframe
frame=    3 fps=0.0 q=-0.0 Lsize=N/A time=00:00:00.08 bitrate=N/A speed=0.0013x    
video:1kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[Parsed_libvmaf_5 @ 0x6478dc0] VMAF score: 14.258492
corrupted double-linked list
qemu: uncaught target signal 6 (Aborted) - core dumped
```

This can be reproduced by building the Docker image from https://github.com/slhck/ffmpeg-quality-metrics/ and running:

```bash
./docker_run.sh test/dist-854x480.mkv test/ref-1280x720.mkv -m vmaf --vmaf-features ciede
```

**Comment by slhck** [comment]
Similarly to https://github.com/Netflix/vmaf/issues/1150#issuecomment-1371006787, this only fails with the linked ffmpeg package. Under macOS and Homebrew ffmpeg/libvmaf it works.

**Comment by kylophone** [comment]
Same as #1103, fyi @nilfm

**Comment by slhck** [comment]
Oh, okay, let's close this one then!

**Comment by nilfm** [comment]
#1164 fixes #1103, and should fix this as well.
